protected/views/site/contact.php

```diff
@@ -1,128 +1,138 @@
+    </div>
 <? } else {
     Yii::app()->captcha->generate();
 ?>
- <div class="content">
-            <div class="container">
-                <section class="page-title-section">
-                    <div class="page-title">
-                        <ol class="breadcrumb pull-right">
-                            <li><a href="/">Home</a></li>
-                            <li class="active">Contact</li>
-                        </ol>
-                        <h1 class="h4">Contact</h1>
-                    </div>
-                </section>
-                <div class="subsection">
-                    <img src="../images/new_interface_image/shekmun_map.png" alt="Map highlighting the GigaDB location on 1 On Kwan Street in Sha Tin, Hong Kong">
+    <div class="content">
+        <div class="container">
+            <section class="page-title-section">
+                <div class="page-title">
+                    <ol class="breadcrumb pull-right">
+                        <li><a href="/">Home</a></li>
+                        <li class="active">Contact</li>
+                    </ol>
+                    <h1 class="h4">Contact</h1>
                 </div>
-                <section>
-                    <div class="row">
-                        <div class="col-xs-9">
-                            <div class="underline-title">
-                                <div>
-                                    <h2 class="h4">Contact form</h2>
+            </section>
+            <div class="subsection">
+                <img src="../images/new_interface_image/shekmun_map.png" alt="Map highlighting the GigaDB location on 1 On Kwan Street in Sha Tin, Hong Kong">
+            </div>
+            <section>
+                <div class="row">
+                    <div class="col-xs-9">
+                        <div class="underline-title">
+                            <div>
+                                <h2 class="h4">Contact form</h2>
+                            </div>
+                        </div>
+                        <div class="subsection">
+                            <p>For more information or questions regarding submitting data to GigaDB, please contact us at: <a href="mailto:database@gigasciencejournal.com" target="_blank">database@gigasciencejournal.com</a>.</p>
+                            <p>Fields with <span>*</span> are required.</p>
+                        </div>
+
+
+
+                        <? $form = $this->beginWidget('CActiveForm', array('htmlOptions' => array('class' => 'form contact-form'))); ?>
+                        <div class="col-xs-7">
+                            <div class="form-group">
+                                <?= $form->labelEx($model, 'name', array('class' => 'control-label')); ?>
+                                <?= $form->textField($model, 'name', array('class' => 'form-control', 'required' => true, 'aria-required' => 'true', 'aria-describedby' => $model->hasErrors('name') ? 'nameError' : null)); ?>
```

Comment:
Updated this (and all subsequent inputs) with the attributes required, aria-required and aria-describedby.

aria-describedby value is set conditionally based on whether error is displayed or not